Like on Linux, don't include swap in memory stats, or else we never get
our duck swimming anywhere near the top (unless the system is heavily
going down).

--- sys_sunos.c
+++ sys_sunos.c
@@ -192,8 +192,8 @@
 	bm.swap_used = cur.lSwapMax - cur.lSwapFree;
 
 	GetMemoryStats(&cur.lMemMax, &cur.lMemFree);
-	bm.mem_max = cur.lMemMax + cur.lSwapMax;
-	bm.mem_used = cur.lMemMax - cur.lMemFree + bm.swap_used;
+	bm.mem_max = cur.lMemMax;
+	bm.mem_used = cur.lMemMax - cur.lMemFree;
 
 	if (dbg_print_mem) {
 		static int i = 0;
--- bubblemon.c
+++ bubblemon.c
@@ -1185,7 +1185,7 @@
 #endif				/* ENABLE_MEMSCREEN */
 #ifdef ENABLE_CPU
     static int yoh;
-    static int avg;
+    static unsigned int avg;
     int hibyte, y, pos;
 
     /* CPU load buffer */
